rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -1,35 +1,32 @@
 import logging
 import os
 import re
-from typing import Any, Dict, List, Optional, Text, Union
-import numpy as np
-
+import scipy.sparse
+from typing import Any, Dict, List, Optional, Text
 from sklearn.feature_extraction.text import CountVectorizer
 from rasa.nlu import utils
 from rasa.nlu.config import RasaNLUModelConfig
-from rasa.nlu.featurizers import Featurizer
+from rasa.nlu.featurizers.featurzier import Featurizer
 from rasa.nlu.model import Metadata
 from rasa.nlu.training_data import Message, TrainingData
-
-logger = logging.getLogger(__name__)
-
 from rasa.nlu.constants import (
-    MESSAGE_RESPONSE_ATTRIBUTE,
-    MESSAGE_INTENT_ATTRIBUTE,
     MESSAGE_TEXT_ATTRIBUTE,
     MESSAGE_TOKENS_NAMES,
     MESSAGE_ATTRIBUTES,
     MESSAGE_SPACY_FEATURES_NAMES,
-    MESSAGE_VECTOR_FEATURE_NAMES,
+    MESSAGE_VECTOR_SPARSE_FEATURE_NAMES,
     SPACY_FEATURIZABLE_ATTRIBUTES,
+    MESSAGE_INTENT_ATTRIBUTE,
+    CLS_TOKEN,
 )
 
+logger = logging.getLogger(__name__)
+
 
 class CountVectorsFeaturizer(Featurizer):
-    """Bag of words featurizer
-
-    Creates bag-of-words representation of intent features
-    using sklearn's `CountVectorizer`.
+    """
+    Creates a sequence of token counts features
+    Based on sklearn's `CountVectorizer`.
```

Comment:
By the style guidelines, there should be one line text on the same line as `"""` for first brief description

Reply:
also the sentence should be only one line: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings